run.sh

```diff
@@ -0,0 +1,120 @@
+#! /usr/bin/env sh
+# Convenience script to run the wikiparser with the maps generator as a drop-in replacement for the scraper.
+#
+# 1. Finds latest map build.
+# 2. Builds wikiparser.
+# 3. Extracts wikidata ids and wikipedia urls from generator intermediate files `id_to_wikidata.csv` and `wiki_urls.txt`
+# 4. Runs wikiparser for all input dump files.
+#
+# For more information on the map generator, see
+# <https://github.com/organicmaps/organicmaps/blob/b52b42bd746fdb8daf05cc048f0b22654cfb9b8e/tools/python/maps_generator/README.md>.
+
+USAGE="./run.sh DUMP_FILE.json.tar.gz [DUMP_FILE.json.tar.gz...]
+
+Environment Variables:
+- MAPS_BUILD_ROOT: The directory with all maps builds, defaults to $HOME/maps_build, ignored if MAPS_DIR is set.
+- MAPS_DIR: The map build directory to use for generating descriptions, defaults to the latest directory in MAPS_BUILD_ROOT."
+
+set -eu
+# set -x
+
+# Write printf-style message to stderr with a timestamp and line ending.
+log () {
+    printf "%s " "$(date '+%Y-%m-%dT%H:%M:%SZ')" >&2
+    # shellcheck disable=2059
+    printf "$@" >&2 # Forward all arguments to printf.
```

Comment:
Can echo be used here?

Reply:
I used printf because echo doesn't handle escaped characters like `\t` and `\n` in a portable way, and you can format numbers and other things nicely.

Reply:
1. Do we really expect newlines and tabs in logs? Is it good practice?
2. What kind of number formatting happens in this line?

Reply:
1. I think newlines and tabs are helpful for separating long content, but I don't think it's a necessity here.
2. On line 25 it wrapper around printf, so any call to log can use printf's formatting string